Bottom: 356a6032447d6c5310176f254b52046d9351985c
Top:    4c1e9af7174f58ced0b5b72be7c3c8278afe6684
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-13 00:44:06 +0100

Apply the default hidnogui HID to the GUIs as well.

This saves them having to re-implement NOOP implementations for
functions the "nogui" HID provides a non "CRASH;" implementation
for.


---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index ea0a0dd..091f472 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -2330,6 +2330,7 @@ hid_gtk_init ()
   ghid_hid.flush_debug_draw         = ghid_flush_debug_draw;
   ghid_hid.finish_debug_draw        = ghid_finish_debug_draw;
 
+  apply_default_hid (&ghid_hid, 0);
   hid_register_hid (&ghid_hid);
 #include "gtk_lists.h"
 }
diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 4f7e12a..c2513bf 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -4065,6 +4065,7 @@ hid_lesstif_init ()
   lesstif_hid.flush_debug_draw        = lesstif_flush_debug_draw;
   lesstif_hid.finish_debug_draw       = lesstif_finish_debug_draw;
 
+  apply_default_hid (&lesstif_hid, 0);
   hid_register_hid (&lesstif_hid);
 #include "lesstif_lists.h"
 }
